sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs

```diff
@@ -224,6 +224,10 @@ internal static string GetDefaultRunId()
 
     internal static void SetReportingUrlAndWorkspaceId()
     {
+        if (ServiceEndpoint == null)
```

Comment:
Should this throw instead of exiting with no action?  When would this be called if `ServiceEndpoint` is null?  Would that be correct?  Is it better to fail silently?  Could we detect it early and throw at the earliest point and then do null suppression here with confidence that it would never be null?

Reply:
This method is internal for testing purposes and is only called from one private method [`InitializePlaywrightServiceEnvironmentVariables`](https://github.com/Azure/azure-sdk-for-net/blob/45bccb4e7273d720060d497897609f0accde7aa3/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs#L211) 

The private method is called from the constructors, both of which do this check and return if null.
- https://github.com/Azure/azure-sdk-for-net/blob/45bccb4e7273d720060d497897609f0accde7aa3/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs#L69-L85

I could do null suppression here instead, that would also be fine.

Reply:
Updated to throw exception.